dry_run/tasks/simple.py

```diff
@@ -0,0 +1,49 @@
+from cloudferrylib.scheduler import task
+from cloudferrylib.utils import utils as utl
+
+
+LOG = utl.get_log(__name__)
+
+
+class AddNumbers(task.Task):
+
+    def __init__(self, a, b):
+        self.a = a
+        self.b = b
+        super(task.Task, self).__init__()
```

Comment:
Please move the `super()` call to the top, you should not rely on what base class is doing.